index.Rmd

```diff
@@ -27,10 +30,92 @@ _Programming skills_
 
 ## Learning Objectives {-}
 
-<!-- Learning objectives for this activity come from the [Genetics Core Competencies](https://genetics-gsa.org/education/genetics-learning-framework/): -->
+- Open RStudio and locate important panels (R console, Help)
+- Run commands in the R console
+- Explain what vectors and dataframes are in R
+- Use common commands to manipulate R objects (e.g. assigning, printing, subsetting, mathematical operations)
+- Carry out a basic exploration of rectangular data using base R commands (e.g. `dim`, `summary`, `str`)
 
-<!-- - Objective 1 -->
-<!-- - Objective 2 -->
-<!-- - Objective 3 -->
+## Core Competencies {-}
+
+This activity addresses the following core concepts and competencies:
```

Comment:
These seem appropriate but take a bit of thought to map onto the exercises.  Beyond the scope of this PR but perhaps worth adding a third column with some breadcrumbs e.g. us_contagious_diseases address V&C #2 and #4 while swirl addresses NIBLSE C7